pkg/param/param.go

```diff
@@ -135,6 +137,21 @@ type KopiaServerCreds struct {
 	ConnectOptions map[string]int
 }
 
+// RepositoryServer contains where to store Repository Server CR Artifacts
```

Comment:
This isn't necessarily true right?
We are going to store the repo server CRs in the etcd itself right?

Reply:
Yeah, so this RepositoryServer struct basically contains the selective artifacts from the Repo Server CR. It is not the Respo Server CR. It's only templating parameters.

Reply:
what are selective artifacts?